### DISCUSSION TITLE
(Fixed) MacOS won't find the app

### DISCUSSION THREAD

**Discussion by Elmerot** [Universal-Debloater-Alliance/universal-android-debloater-next-generation#725] [Q&A]
EDIT: Following [these steps](https://www.wikihow.com/Open-Applications-Using-Terminal-on-Mac) worked. 

Hi,
and first, thank you for setting this thing up!
I have done everything in the wiki and getting started instrux, up to initialising the uad-ng-macos software on my Macbook with OS 14.7, and it will not open with `open -a uad-ng-macos` in the terminal. Is there any other way to initialise this?
(Terminal only prints "Unable to find application named 'uad-ng-macos'".)
Thank you!

**Comment by AnonymousWP** [comment]
Great to hear! Closing this discussion.
